src/olympia/reviewers/utils.py

```diff
@@ -879,11 +881,15 @@ def process_sandbox(self):
         self.set_files(amo.STATUS_DISABLED, self.files,
                        hide_disabled_file=True)
 
-        # Unset needs_human_review on the latest version - it's the only
-        # version we can be certain that the reviewer looked at.
         if self.human_review:
+            # Unset needs_human_review on the latest version - it's the only
+            # version we can be certain that the reviewer looked at.
             self.version.update(needs_human_review=False)
```

Comment:
This isn't related to this patch, but shouldn't we also reset the `needs_human_review_by_mad` (reviewerflags)?

Reply:
Probably, yes. Could be fixed when doing https://github.com/mozilla/addons-server/issues/14943 (even though we want it for listed as well)